### PR TITLE
Fix failed request cookies

### PR DESCRIPTION
**Pull request by ThisIsOstad**
Related issue: https://github.com/Cuadrix/puppeteer-page-proxy/issues/36

### CODE REVIEW

File: src/lib/cookies.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const formatCookie = (cookie) => {
 class CookieHandler extends CDP {
     constructor(request) {
         super(request._client);
-        this.url = request.isNavigationRequest() ? request.url() : request.frame().url();
+        this.url = request.isNavigationRequest() ? request.url() : request.frame()?.url() || request.url();
```
Comment (ardalann) [inline src/lib/cookies.js:67]: I don't think this works. Example: https://codepen.io/ardalann/pen/RwogYdj?editors=1111

**Review by ardalann — CHANGES_REQUESTED** [review]
Almost there!

P.S. I love the PR review experience on Github lol

File: src/lib/cookies.js (hunk 1 above)
Comment (ardalann) [reply src/lib/cookies.js:67]: UPDATE: My example was wrong! this works: https://codepen.io/ardalann/pen/RwogYdj?editors=1111

**Comment by wynnfarm** [comment]
Hello, I am seeing 4 issues related to this bug.  Is this ready to be merged back into the main project and a new version published?

**Pull request merged**